zipkin-ui/package.json

```diff
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "dev": "node node_modules/webpack-dev-server/bin/webpack-dev-server.js --hot --inline",
-    "test": "node node_modules/karma/bin/karma start --single-run",
+    "test": "NODE_ENV=production node node_modules/karma/bin/karma start --single-run",
     "test-watch": "node node_modules/karma/bin/karma start",
     "lint": "node node_modules/eslint/bin/eslint js test",
-    "build": "node node_modules/webpack/bin/webpack.js --bail"
+    "build": "NODE_ENV=production node node_modules/webpack/bin/webpack.js --bail"
```

Comment:
It's maybe easier to just add `--optimize-minimize --optimize-dedupe` here, to the CLI arguments, then we don't need to tweak the webpack config or set environment variables etc.

Passing environment variables to the script like this will also not work on Windows, so we'd have to use crossenv or something like that.